### PR TITLE
Attempt to make the wordlists a little less "invasive"

### PR DESCRIPTION
**Pull request by purifiedfr**
See commits for the list of words I have removed, these are mostly the words mentioned in #546, specifically [this comment](https://github.com/BeamMP/BeamMP/pull/546#issuecomment-2267098843).
Other suggestions of words to remove would be welcome, and I'll try to add more commits to this to remove even more words.

Note: some words in the diff are removed and then added again, I might've messed something up but the words have been kept. They're in CRLF, not sure if that's causing any issues.

### CODE REVIEW

File: lua/ge/extensions/multiplayer/filters/inappropriate.txt [hunk 1]
```diff
@@ -5,15 +5,12 @@ acrotomophilia
 alabama hot pocket
 alaskan pipeline
 anal
-anal
 anilingus
 anus
-anus
 apeshit
 arab
```
Comment (Starystars67) [inline lua/ge/extensions/multiplayer/filters/inappropriate.txt]: Remove this one please.

File: lua/ge/extensions/multiplayer/filters/inappropriate.txt [hunk 2]
```diff
@@ -91,7 +87,6 @@ carpet muncher
 carpetmuncher
 catholic
```
Comment (Starystars67) [inline lua/ge/extensions/multiplayer/filters/inappropriate.txt]: Remove this one please.

**Review by Starystars67 — COMMENTED** [review]
Nice work, thank you for getting around to this one for us!

If you could address the minor comments I added that would be great thank you!

**Comment by purifiedfr** [comment]
GitHub seems to show these in a strange way for me, for clarification, you mean lines 13/11 and 92/88, right?

**Comment by Starystars67** [comment]
Yes, the removal of catholic and anus.

**Comment by purifiedfr** [comment]
They should now be removed.

File: lua/ge/extensions/multiplayer/filters/inappropriate.txt (hunk 1 above)
Comment (Starystars67) [reply lua/ge/extensions/multiplayer/filters/inappropriate.txt]: please remove arab too.

**Review by Starystars67 — CHANGES_REQUESTED** [review]

**Comment by purifiedfr** [comment]
I'll also try find other words (such as cancer) and remove those.

**Review by Starystars67 — APPROVED** [review]

**Comment by Starystars67** [comment]
Happy to go ahead and merge if you are ready!

**Comment by purifiedfr** [comment]
Wordlists should be ready to merge.

**Pull request merged**